en/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: news_post
+title: "Ruby 2.4.0-preview2 Released"
+author: "naruse"
+translator:
+date: 2016-09-08 18:00:00 +0900
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.4.0-preview2.
+
+Ruby 2.4.0-preview2 is the second preview of Ruby 2.4.0.
+This preview2 is released to get feedback from community.
+Feel free to
+[send feedback](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+since you can still change the features.
+
+## [Unify Fixnum and Bignum into Integer](https://bugs.ruby-lang.org/issues/12005)
+
+Though [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+doesn't specify details of the Integer class,
+CRuby had two visible Integer classes: Fixnum and Bignum.
```

Comment:
Why use `CRuby` here? Usually we only say `Ruby` on this site, MRI/CRuby is always assumed;
also "CRuby has foo but Ruby 2.4 has bar" is illogical.

Maybe "Prior versions of Ruby had two visible ..." ?